src/networking.c

```diff
@@ -2920,7 +2920,7 @@ NULL
     }
 }
 
-/* HELLO [<protocol-version> [AUTH <user> <password>] [SETNAME <name>] ] */
+/* HELLO [<protocol-version>] [AUTH <user> <password>] [SETNAME <name>] ] */
```

Comment:
The brackets is not closed. I think the old one is correct.

Reply:
My bad.